docs/current_docs/manuals/administrator/tutorials/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+slug: /manuals/administrator/tutorials
+displayed_sidebar: "current"
+---
+
+# Tutorials
+
+Learn more about Dagger using the step-by-step walkthroughs in this section.
+
```

Comment:
```suggestion
```

The tutorials section can be removed - see other comments for what to do with each of the 3 articles.

Reply:
Done